combining_caribou_teeth.R

```diff
@@ -1,17 +1,20 @@
-sample01A <- read.csv("01A.csv", header = FALSE)
-colnames(sample01A) <- c("time", "Sr")
 
-sample01B <- read.csv("01B.csv", header = FALSE)
-colnames(sample01B) <- c("time", "Sr")
+sampler <- function(fileNumber, fileCharacter) {
+  fileName <- sprintf("%02d%s.csv", fileNumber, fileCharacter)
```

Comment:
This string probably looks like magic. It continues to use "string fromatting" (I've mentioned this before).  The `%s` is replaced by a string or a character (in this case the file character [ABC]).  `%d` replaces with an integer, but if we did that our files would have to be named 1A.csv and not 01A.csv.  To fix this we "zero pad" the string.  We tell the string formatter that we want our full string to be 2 characters in length and that the padded characters should be 0's `%02d`.  If you wanted them to be hyphens it would look like `%-2d` (not that you'd want to, just wanted to show that this is fungible.